### PR TITLE
Skipgram works with CUDA (incl. evaluate skipgram)

### PR DESCRIPTION
**Pull request by tharangni**

### CODE REVIEW

File: practical-2/evaluate_skipgram.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
         # print("Words skipped {}".format(skipped_count))
     print("Number of unique words in the file: ", len(existing_words))
 
-    with open('skipgram_lst.out', 'w+') as f:
+    with open('lst.out', 'w+') as f:
```
Comment (samarthbhargav) [inline practical-2/evaluate_skipgram.py]: can you change this back?

File: practical-2/skipgram.py [hunk 2]
```diff
@@ -43,6 +43,7 @@ def __init__(self, vocab, embedding_dim, use_cuda=False):
         self.use_cuda = use_cuda
 
         if use_cuda:
+            print("sam, fuck you. i made it work with cuda :P.")
```
Comment (samarthbhargav) [inline practical-2/skipgram.py]: I'm going to link this in the report, I swear :laughing:

File: practical-2/evaluate_skipgram.py (hunk 1 above)
Comment (tharangni) [reply practical-2/evaluate_skipgram.py]: ok

File: practical-2/skipgram.py (hunk 2 above)
Comment (tharangni) [reply practical-2/skipgram.py]: be my guest :P

**Pull request merged**